src/components/PostsList/PostsList.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Post } from '../../types/Post';
+
+type Props = {
+  posts: Post[];
+  postId: number | null;
+  onPost: (id: number | null) => void;
+};
+
+export const PostsList: React.FC<Props> = ({
+  posts,
+  postId,
+  onPost,
+}) => {
+  const handleSetOpened = (id: number) => {
+    return (postId !== id)
```

Comment:
The handleSetOpened function is currently returning the result of a function call. However, it should not return anything. Instead, it should only call the function. Consider removing the `return` keyword.